narwhal/primary/src/primary.rs

```diff
@@ -555,13 +759,55 @@ impl PrimaryToPrimary for PrimaryReceiverHandler {
         &self,
         request: anemo::Request<PrimaryMessage>,
     ) -> Result<anemo::Response<()>, anemo::rpc::Status> {
-        let message = request.into_body();
-        self.tx_primary_messages
-            .try_send(message)
+        let PrimaryMessage::Certificate(certificate) = request.into_body();
+        self.tx_certificates
+            .try_send((certificate, None))
             .map(|_| anemo::Response::new(()))
             .map_err(|e| anemo::rpc::Status::internal(e.to_string()))
     }
 
+    async fn request_vote(
+        &self,
+        request: anemo::Request<RequestVoteRequest>,
+    ) -> Result<anemo::Response<RequestVoteResponse>, anemo::rpc::Status> {
+        // TODO: Remove manual code for tracking inflight requests once Anemo issue #9 is resolved.
+        let author = request.body().header.author.to_owned();
```

Comment:
While we can add rate-limiting on the server side, i don't think we want to remove the application level limiting, in that we want to enforce that narwahl is only trying to get votes for 1 round at a time

Reply:
#9 requests both rate limiting and inflight limiting as Anemo features. So the inflight limiting on Anemo side is what would enable this to be removed here. Agree rate limiting would be insufficient